v4/source/status.yaml

```diff
@@ -0,0 +1,14 @@
+  /users/{user_id}/status:
+    get:
+      tags:
+        - status
+      summary: Get user status by id
+      description: |
+        Get user status by id from the server.
+        ##### Permissions
+        No permission required.
+      responses:
+        '200':
+          description: User status retrieval successful
+          schema:
+            type: string
```

Comment:
Can you update this to be a ref to the Status object? Might need to create it in definitions if it doesn't exist